agreement.md

```diff
@@ -0,0 +1,151 @@
+# Aragon Network Agreement
+
+## Section 0. Background
+The Aragon Network is a Decentralized Autonomous Organization that is building a dispute resolution protocol and other smart contract infrastructure.
+
+The Aragon Network is governed by Aragon Network Token (ANT) holders, who supported the initial creation of the network and under this Agreement will continue to guide its evolution through established governance processes.
+
+This Agreement exists to enumerate the shared mission, principles, and governance guidelines that bind ANT holders in their collaborative effort to create value for the Aragon Network.
+
+## Section 1. RFC 2119 compliance
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED",  "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119.
+
+## Section 2. Immutable guidelines
+The guidelines in this section SHALL NOT be amended. If any other guidelines in this Agreement conflict with these immutable guidelines, the immutable guidelines SHALL take priority for enforcement purposes. These guidelines apply to all proposals submitted to the Aragon Network organization.
+
+1. Proposals MUST be governed by this Agreement, or a future version of this Agreement as modified within the bounds of the Agreement.
+
+2. Disputes over a proposal relying on this Agreement MUST be resolved by Aragon Court.
+
+3. Proposals MUST NOT disproportionately benefit a majority of ANT holders voting on a proposal over the minority.
+
+4. Proposals MUST NOT attempt to cast a vote using ANT held in the address of any versions of the Agent app installed on the Aragon Network organization, or using ANT held in any Network contract that fulfills similar functionality as the Agent app.
+
+5. Proposals to amend either the current Aragon Court Controller contract or any new Controller contract that may be proposed for Aragon Court MUST NOT amend Aragon Court configurations, funds, and modules governors to be any entity other than an Agent app installed on the Aragon Network organization.
+
+6. If a proposal deploys a new Aragon Court Jurors Registry contract and/or amends Aragon Court to use a new Jurors Registry, then the new Jurors Registry MUST use ANJ as the native juror token.
+
+7. Proposals MUST NOT activate in Aragon Court any ANJ controlled by the Aragon Network.
+
+8. Proposals MUST NOT use the Aragon Network organization’s role as Aragon Court funds governor to transfer ANJ staked and/or activated by Aragon Court jurors to another Ethereum account.
+
+## Section 3. Mutable guidelines
+The guidelines in this section MAY be amended. These guidelines apply to all proposals put forth to the Aragon Network organization.
+
+1. Proposals MUST be consistent with the Aragon Manifesto.
+
+2. Proposals to transfer assets from the Aragon Network organization to the exclusive control of any other entity SHALL NOT be allowed until Ethereum block #15,000,000.
```

Comment:
Thinking more about this guideline, I think it could complicate a bit the project's path to decentralization. One of the concentrated powers in Aragon right now is the economic power (which is a very strong/influential power). This power is concentrated in one entity: the Aragon Association (and whoever controls that legal entity). If the Aragon Network DAO becomes self-sustainable, with a treasury of its own, and the DAO is able to use that treasury for the development of the Aragon Network, there will be another source of funding for the Aragon Network. At this moment there could be said that the economic power in the AN DAO is decentralized and making the decentralization of the DAO more powerful. Thus, even though I understand the point of this guideline, the sooner the DAO is self-sustainable and able to spend its own treasury on the Aragon Network development, without depending solely on the Aragon Association, the better.